multicluster/charts/linkerd-multicluster-link/templates/service-mirror.yaml

```diff
@@ -58,7 +58,7 @@ rules:
     verbs: ["list", "get", "watch"]
   - apiGroups: ["multicluster.linkerd.io"]
     resources: ["links/status"]
-    verbs: ["update"]
+    verbs: ["update", "patch"]
```

Comment:
Do you know if we still require `update`  here?

Reply:
I don't believe so, but it might be safer to keep it to give us more flexibility in terms of the exact API calls that are used to update the status.